clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/client/model/Task.java

```diff
@@ -40,6 +40,7 @@ public Task(com.netflix.titus.grpc.protogen.Task grpcTask) {
     startedAt = getTimestampFromStatus(grpcTask, TaskStatus.TaskState.StartInitiated);
     finishedAt = getTimestampFromStatus(grpcTask, TaskStatus.TaskState.Finished);
     containerIp = grpcTask.getTaskContextOrDefault("task.containerIp", null);
+    agentId = grpcTask.getTaskContextOrDefault("agent.instanceId", null);
```

Comment:
Guessing deck would be doing the lookup of the IP based on this agent Id?

Hopefully the long term solution comes sooner rather than later

Reply:
Yes deck will do the lookup. They are targeting Q2. It was supposed to be end of Q1 but it is delayed from COVID wfh shifting some priorities.